assets/pages/new-contributions-landing/contributionsLanding.scss

```diff
@@ -332,6 +332,124 @@ footer[role="contentinfo"] a {
 
 }
 
+// Alternative form layout 
+.gu-content--contribution-form-two {
+  header[role="banner"] {
+
+    @include mq($from: tablet) {
+      height: 120px;
+    }
+  }
+
+  .guardian-logo {
+    width: 100px;
+
+    svg {
+      width: 100%;
+    }
+
+    @include mq($from: tablet) {
+      width: auto;
+    }
+  }
+
+
+  .gu-content__main {
+    background-color: gu-colour(sport-faded);
+    display: block; // only here to cancel the display: flex
+  }
+
+  .gu-content__content {
+    margin: 0 auto;
+    max-width: 1015px;
+    border: none;
+    padding: 0;
+    background-color: transparent;
+    
+    width: 100%;
+
+    &--flex {
+      display: flex;
+      justify-content: space-between;
+      flex-direction: row-reverse;
+    }
+
+    @include mq($from: tablet) {
+      margin-left: $content-left-margin-tablet
+    }
+
+    @include mq($from: leftCol) {
+      margin-left: $content-left-margin-leftCol
+    }
+  }
+
+  .gu-content__form {
+    margin: auto;
+    width: auto;
+    padding: 0 10px;
+    border-color: gu-colour(brightness-86);
+    border-style: none solid;
+    border-width: 0 1px;
+    border-bottom: 1px solid gu-colour(brightness-86);
+    background-color: gu-colour(brightness-100);
+
+    @include mq($from: tablet) {
+      width: 380px;      
```

Comment:
can this use the nearest `gu-span(number)` equivalent? (that's the columns in the grid system mixin)